src/zos-getentropy.cc

```diff
@@ -21,13 +20,13 @@
 #endif
 
 static unsigned char _value(int bit) {
-  unsigned long long t0, t1, start;
+  unsigned long long t0, t1 = 0, start;
   int i;
-  asm(" la 15,0 \n svc 137\n" ::: "r15", "r6");
+  __asm volatile(" la 15,0 \n svc 137\n" ::: "r15", "r6");
```

Comment:
just a tip/nitpick.  You don't need to use `__asm` in the code of the library.

Reply:
Thanks. I'll make all those replacements in a separate PR.